darshan-util/pydarshan/darshan/cli/summary.py

```diff
@@ -458,6 +459,20 @@ def register_figures(self):
                 )
                 self.figures.append(opcount_fig)
 
+        #########################
+        # Data Access by Category
+        if not {"POSIX", "STDIO"}.isdisjoint(set(self.report.modules)):
+            data_access_by_cat_fig = ReportFigure(
+                section_title="Data Access by Category",
+                fig_title="",
+                fig_func=data_access_by_filesystem.plot_with_report,
+                fig_args=dict(report=self.report, num_cats=8),
+                fig_description="",
```

Comment:
Feel free to suggest content for these fields, and a placeholder when the figure cannot be produced if you want that.

More pressing at the moment is the inability to resize the figure through `ReportFigure` `fig_width` though I think.

Reply:
> there's some discussion from Shane in the matching issue about:
> 
>     * how to pick the top `N` categories vs. the magic number I've used here

I think as long as the current code chooses the top N categories (in terms of total read/write volume) for displaying, we're good. 8 is a reasonable enough value for me -- if more than 8 mount points are found in the log, though, we just want to make sure the 8 most important are displayed. From a quick scan of the code I think this is the case, so good to go.

> 
>     * something about ditching the plot entirely and using html table instead?

Can probably just ignore this. My initial concern was how exactly we could embed tables into the HTML report (mostly just wanting to avoid generating tables with our own HTML code if possible) -- looks like all the tables are generated as mpl figures, which is fine by me.

Reply:
For title: "Data access by category"

For description: "Provides access details for the top data access categories, in terms of cumulative read/write volume. Categories are shown in descending order of total I/O volumes. Possible data access categories currently include file system mount points and standard streams accessed by the application."

@carns might have some better ideas on how to tweak the language. "categories" is kind of a vague term that's hard to work with. But, we need a term vague enough to encapsulate mount points, standard streams, and even things like DAOS object storage pools -- anything a user may read/write from/to basically.

Reply:
I'm good with that title.  Maybe simplify the description/caption, though?

"Summary of data access volume categorized by storage target (e.g., file system mount point) and sorted by volume."

We can expand on that later if we add other category types, but it's probably best to keep it simple for now.  The "Standard streams" category arguably isn't really a storage target, but I think people will get it.